### PR TITLE
Add 2023 commissioning logbook entries

### PR DESCRIPTION
**Pull request by awegsche**
- some minor corrections to existing entries
- added days 04-06 (validation of new 120cm corrs), 04-10, 04-16 (both Ions)

### CODE REVIEW

File: docs/resources/logbook/2023_lhc.md
Comment (JoschD) [inline docs/resources/logbook/2023_lhc.md:32]: These link to the OP logbook. I am getting "could not retrieve selected events"

File: docs/resources/logbook/2023_lhc.md
Comment (fsoubelet) [inline docs/resources/logbook/2023_lhc.md:49]: Blunders and stuggles to get to the end of the ramp are irrelevant. What matters for the tooltip is only what was observed and achieved in the shift.

File: docs/resources/logbook/2023_lhc.md
Comment (fsoubelet) [inline docs/resources/logbook/2023_lhc.md:37]: This link directs to a logbook entry with measurements of the kickgroup. Is there not a better one?

**Review by fsoubelet — CHANGES_REQUESTED** [review]
Needs links fixes and some clarifications. Table formatting is messed up but that's an easy fix for later.

File: docs/resources/logbook/2023_lhc.md
Comment (awegsche) [reply docs/resources/logbook/2023_lhc.md:37]: No, that shift was not well documented

File: docs/resources/logbook/2023_lhc.md
Comment (awegsche) [reply docs/resources/logbook/2023_lhc.md:49]: Ok

File: docs/resources/logbook/2023_lhc.md
Comment (JoschD) [reply docs/resources/logbook/2023_lhc.md:32]: the id is 1081 (OMC logbook) not 322 (OP logbook)

**Comment by awegsche** [comment]
So, problems fixed.

we could optionally change the logbook entry for the first ion shift and it should be good to go

**Comment by awegsche** [comment]
> 

I don't know what you mean by "table formatting messed up" for me it displays perfectly

**Review by fsoubelet — APPROVED** [review]
I meant the formatting of the table code.
I fixed a link that pointed to the wrong day (ramp meas summary) and tweaked the tooltips for ion optics entries.

All links work fine. I'm approving but I'd like to get approval from @JoschD too before we merge.

**Review by JoschD — APPROVED** [review]

**Pull request merged**
